helix-rest/src/main/java/org/apache/helix/rest/server/service/InstanceServiceImpl.java

```diff
@@ -213,7 +219,12 @@ public StoppableCheck getInstanceStoppableCheck(String clusterId, String instanc
           instancesForNextCheck.add(instance);
         }
       } catch (InterruptedException | ExecutionException e) {
-        LOG.error("Failed to get StoppableChecks in parallel. Instance: {}", instance, e);
+        LOG.warn("Failed to get StoppableChecks in parallel. Instance: {}. Caused by {}", instance,
```

Comment:
Why it is a warning now?

Reply:
I think the failure reason does not really matter here (although an Exception looks to be a problem to me more than a warning).
The real matter thing is that whether the result is complete and fully computed. If not certain, then this is an error.

Reply:
The code change is not necessary and kept as it is.

Regarding warn or error, I look at it from server perspective: if the exception is not a server exception that affects running of the REST server, I would say it is an error.
But obviously this is not an exception that stops REST server running. It is just a normal running of the API check. It should not be logged as an error. A warning should already be enough to help debug.